src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -301,8 +301,10 @@ void mp::LXDVMImageVault::remove(const std::string& name)
 {
     try
     {
-        lxd_request(manager, "DELETE",
-                    QUrl(QString("%1/virtual-machines/%2").arg(base_url.toString()).arg(name.c_str())));
+        auto task_reply = lxd_request(
+            manager, "DELETE", QUrl(QString("%1/virtual-machines/%2").arg(base_url.toString()).arg(name.c_str())));
+
+        lxd_wait(manager, base_url, task_reply, 300000);
```

Comment:
Two things I am wondering about this:
  - is the same timeout appropriate for all lxd waits in this file? If so, it should probably be a constant.
  - OTH, isn't 5min too much to wait for instance deletion? Wouldn't it be reasonable to error out after 2min?

Reply:
> is the same timeout appropriate for all lxd waits in this file? If so, it should probably be a constant.

Right, should make any timeout values constants.

> OTH, isn't 5min too much to wait for instance deletion? Wouldn't it be reasonable to error out after 2min?

Yeah, it's probably overkill in case something did hand when trying to delete.

Reply:
Ok, fixed these.